src/cmd-build

```diff
@@ -242,9 +242,17 @@ img_base=tmp/${imageprefix}-base.qcow2
 # forgive me for this sin
 checksum_location=$(find /usr/lib/coreos-assembler-anaconda/ -name '*CHECKSUM' | head -1)
 
+build_image() {
+	local size
+	size="$(python3 -c 'import sys, yaml; print(yaml.load(sys.stdin)["size"])' < "$configdir/image.yaml")G"
+	qemu-img create -f qcow2 "$workdir/diskimage.qcow2" "$size"
```

Comment:
Instead of `$workdir` this should be in the build tmpdir `$(pwd)` (which would require passing it as an argument to `runvm` for cleanliness)

Reply:
In that case might as well pass the base image path directly to runvm, right?